packages/anoncreds/src/AnonCredsApi.ts

```diff
@@ -148,20 +204,21 @@ export class AnonCredsApi {
       }
     }
 
-    const credentialDefinition = await this.anonCredsIssuerService.createCredentialDefinition(this.agentContext, {
-      issuerId: options.credentialDefinition.issuerId,
-      schemaId: options.credentialDefinition.schemaId,
-      tag: options.credentialDefinition.tag,
-      supportRevocation: false,
-      schema: schemaResult.schema,
-    })
+    const { credentialDefinition, credentialDefinitionPrivate, keyCorrectnessProof } =
```

Comment:
I was thinking that we can combine the CredentialDefinitionPrivate and the KeyCorrectnessProof. They're always used together right?

Reply:
Note from Ariel: they're not used together.

`keyCorrectnessProof` is used when creating an offer, the `credentialDefinitionPrivate` is used when creating the credential.